target/linux/mpc85xx/image/Makefile

```diff
@@ -126,6 +126,23 @@ define Device/ocedo_panda
 endef
 TARGET_DEVICES += ocedo_panda
 
+define Device/ecw7210
```

Comment:
Look for guidance on proper name above, like compatible, just with "_" instead of comma.